lib/mayaUsd/ufe/UsdPointInstanceModifierBase.h

```diff
@@ -153,23 +185,58 @@ class MAYAUSD_CORE_PUBLIC UsdPointInstanceModifierBase
 
         const size_t instanceIndex = static_cast<size_t>(_instanceIndex);
 
-        PXR_NS::UsdAttribute usdAttr = _getOrCreateAttribute();
-        if (!usdAttr) {
-            return false;
+        // Once setValue() is called, any point instance batch under
+        // construction is closed.  If no batch was created, make a trivial,
+        // unshared batch of a single point instance.
+        _closeBatch();
+
+        PXR_NS::UsdAttribute usdAttr;
+
+        const bool reader = _batch->isReader();
+        const bool writer = _batch->isWriter();
+        _batch->count++;
+
+        if (reader || writer) {
+            usdAttr = _getOrCreateAttribute();
+            if (!usdAttr) {
+                return false;
+            }
         }
 
-        PXR_NS::VtArray<UsdValueType> usdValues;
-        if (!usdAttr.Get(&usdValues, usdTime)) {
-            return false;
+        if (reader) {
```

Comment:
Point instance modifier in the batch that is first to set a value will read into the batch's VtArray.